content/blog/2017/color.md

```diff
@@ -284,14 +284,14 @@ In the end, we loved OddBird orange too much to cut it altogether, and
 we snuck it into the OddSite design. Can you find it? Hint: scroll back
 to the top and try refreshing the page.
 
-As always, we want to know what your favorite web tools are for playing
-with color. Have you tried Xd? What did you think? We’ll be discussing
-this and more in our [OddFriends Slack channel]. Join us!
```

Comment:
Should we add a Twitter link to this one, since it's not specific to discussion of the redesign?

Reply:
Good idea.